run.sh

```diff
@@ -8,11 +8,13 @@
 # https://github.com/0x5c/quick-bot-no-pain
 
 
-# If $BOTENV is not defined, default to 'botenv'
 if [ -z "$BOTENV" ]; then
-    BOTENV='botenv'
+    BOTENV='./botenv'
 fi
 
+if [ -z "$VIRTUAL_ENV" ]; then
+    VIRTUAL_ENV="$BOTENV"
```

Comment:
This should probably be the inverse definition, since there could be something else down the line that (mistakenly?) uses the venv standard var.